files/mailer.rb

```diff
@@ -104,10 +108,10 @@ def handle
           body    body
         end
 
-        puts 'mail -- sent alert for ' + short_name + ' to ' + mail_to.to_s
+        notify 'mail -- sent alert for ' + short_name + ' to ' + mail_to.to_s
```

Comment:
I agree we shouldn't be using puts directly.

There is already a `log` method used in other places, can you re-use that?